eq-author/src/App/dataSettings/SupplementaryDataPage/schemaVersionsTable.js

```diff
@@ -68,7 +68,7 @@ const SchemaVersionTable = ({ surveyId, linkSupplementaryData }) => {
 
   return (
     <>
-      <Title>Datasets for survey ID {surveyId}</Title>
+      <Title>Datasets schema for survey ID {surveyId}</Title>
```

Comment:
```suggestion
      <Title>Dataset schemas for survey ID {surveyId}</Title>
```
Updates the text based on the ticket. The test file will also require changes to reflect this.